crates/sui-json-rpc/src/read_api.rs

```diff
@@ -850,6 +850,21 @@ impl ReadApiServer for ReadApi {
             has_next_page,
         })
     }
+
+    async fn query_objects(
+        &self,
+        _query: SuiObjectResponseQuery,
+        _cursor: Option<ObjectID>,
+        _limit: Option<usize>,
+        _descending_order: Option<bool>,
+        _at_checkpoint: Option<CheckpointId>,
+    ) -> RpcResult<ObjectsPage> {
+        return Err(anyhow!(UserInputError::Unsupported(
+            "queryObject API is not supported on the full node. Please call the indexer."
```

Comment:
indexer is an intra-mysten concept, let's use `enhanced fullnode`

Reply:
This is moved over to extended now